tools/releases/distros.sh

```diff
@@ -105,7 +104,7 @@ function create_tarball() {
   get_ebpf_programs "$arch" "$system" "$kuma_dir"
 
   artifact_dir=$(artifact_dir "$arch" "$system")
-  cp -p "$artifact_dir/envoy/envoy-$ENVOY_VERSION-$envoy_distro" "$kuma_dir/bin"
```

Comment:
Now we are going to pack the envoy binary as `envoy` instead of `envoy-version-distro`. WDYT?

Reply:
that make sense for me. Packing it with envoy-version-distro was a mistake